Slider/Assets/Scripts/Utility/FlashWhite.cs

```diff
@@ -38,5 +39,8 @@ private IEnumerator _Flash(int n)
 
             yield return new WaitForSeconds(0.25f);
         }
+
+        if(callback !=null)
+            callback?.Invoke();
```

Comment:
doesnt the `?.` operator alr check if its null

Reply:
yes 💀